tool/tctl/common/auth_command.go

```diff
@@ -680,12 +680,25 @@ func (a *AuthCommand) generateUserKeys(ctx context.Context, clusterAPI auth.Clie
 	}
 	key.TrustedCA = auth.AuthoritiesToTrustedCerts(hostCAs)
 
+	kubeTLSServerName := ""
```

Comment:
Nit: Move this var definition below, to where `kubeTLSServerName` is set (before you check for multiplex mode).